lib/utils/FileCache.js

```diff
@@ -28,6 +28,13 @@ class FileCache {
 		this._hashOfConfig = hashOfConfig;
 	}
 
+	/**
+	 * @param {string} hashOfConfig
+	 */
+	set hashOfConfig(hashOfConfig) {
+		this._hashOfConfig = hashOfConfig;
+	}
```

Comment:
[suggest] How about adding the following method instead of the setter method?

```js
/**
 * @param {import('stylelint').Config} config
 */
calcHashOfConfig(config) {
	const stylelintVersion = pkg.version;
	const configString = JSON.stringify(config || {});
	this._hashOfConfig = hash(`${stylelintVersion}_${configString}`);
}
```

This suggestion aims to make it easier to find the location of the calculation logic.

Reply:
Thanks. I think your suggestion is great. So, I added `calcHashOfConfig`. [`ca2fb53`](https://github.com/stylelint/stylelint/pull/6356/commits/ca2fb534ba93e2dba40092945e6f56758c99a97e#diff-83240954c077d79828b7a92f75b9e1c412d2b228b367c5864011f57b937fd26e)